stdlib/format.ml

```diff
@@ -954,20 +857,21 @@ let default_pp_print_close_tag = ignore
    Other fields get reasonable default values. *)
 let pp_make_formatter f g h i j =
   (* The initial state of the formatter contains a dummy box. *)
-  let pp_queue = make_queue () in
+  let pp_queue = Queue.create () in
   let sys_tok =
-    make_queue_elem (size_of_int (-1)) (Pp_begin (0, Pp_hovbox)) 0 in
-  add_queue sys_tok pp_queue;
-  let sys_scan_stack =
-    Scan_elem (1, sys_tok) :: scan_stack_bottom in
+    { size = Size.unknown; token = Pp_begin (0, Pp_hovbox); length = 0 } in
+  Queue.add sys_tok pp_queue;
+  let scan_stack = Stack.create () in
+  initialize_scan_stack scan_stack;
```

Comment:
Nitpick:

```ocaml
let scan_stack = initialize_scan_stack (Stack.create ())
```

would be nicer, as it does not give a name to an invalid stack. (One could also have a function for this.)

Reply:
I think this works nice at this particular call site, but doesn't work as well at the other two call sites, where `initialize_scan_stack` is used to re-initialize an existing stack.

Reply:
I agree that it wouldn't work for other use-sites of `initialize_scan_stack`, my point was that `let scan_stack = Stack.create ()` gives a stack name to something that is not a valid stack. (This minor glitch does not occur in the other callsites of `initialize`.)